### ISSUE TITLE
DeckGL keyboard navigation

### ISSUE THREAD

**Issue by hardysabs2** [visgl/deck.gl#4412]
In may application I can tab to ReactMapGL and manouver the map using the keyboard but I wondered if there was any way for DeckGL layers to listen to the keyboard and for me to implement some keyboard navigation there, in particular IconLayer.

If you can direct me to some documentation I'd be grateful.

**Comment by MehdiSaffar** [comment]
I believe you can implement your own login by subclassing MapController and passing an instance to `controller` prop of `DeckGL`

**Comment by Pessimistress** [comment]
deck.gl implements keyboard navigation similar to react-map-gl. You should be able to use keyboard on the [examples](https://deck.gl/#/examples/core-layers/scatterplot-layer).
